### DISCUSSION TITLE
Have match exec cli return a failure on a bad trigger

### DISCUSSION THREAD

**Discussion by bhansconnect** [espanso/espanso#1630] [Feature requests and Ideas]
Currently if you send a trigger to the match exec cli that is not defined, the script will just do nothing. It would be nice if it could also exit with a nonzero exit code. That way you could recognize the failure and do something in response.

So if I have `:espanso` -> `hello world` as my only trigger and I run `espanso match exec -t ":somethingWrong"` it would give me a non-zero exit code.

Then It would be possible to make scripts that run a trigger or print out the original text, for example:

`espanso match exec -t $1 || echo $1`

**Comment by smeech** [comment]
This could, indeed, be useful.

At the moment, the best you get is a warning in `espanso log` such as: `22:18:46 [worker(771475)] [WARN] received match exec request, but no matches have been found for the given query.`
